onnx/defs/text/defs.cc

```diff
@@ -33,6 +29,34 @@ ONNX_OPERATOR_SET_SCHEMA(
                 *ctx.getOutputType(0)->mutable_tensor_type()->mutable_shape());
         }));
 
+static const char* RegexFullMatch_doc =
+    R"DOC(RegexFullMatch performs a full regex match on each element of the input tensor. If an element fully matches the regex pattern specified as an attribute, the corresponding element in the output is True and it is False otherwise. [RE2](https://github.com/google/re2/wiki/Syntax) regex syntax is used.)DOC";
```

Comment:
I wonder what we should do about changes in RE2 over time? Should we specify a specific version of RE2? In the future, if it evolves, we could potentially add a version attribute to keep evolving.

Reply:
As we discussed, I don't think RE2 has explicit versioning unfortunately but the syntax looks very stable. It is also the syntax used by [tensorflow](https://www.tensorflow.org/api_docs/python/tf/strings/regex_full_match) and [PyTorch](https://pytorch.org/text/stable/transforms.html#regextokenizer) regex ops.